### DISCUSSION TITLE
Testing changes without modifying library

### DISCUSSION THREAD

**Discussion by Bootjewolf** [beetbox/beets#4541] [Q&A] [answered]
So maybe this is an easy question to answer. How would I go about testing the changes I make to the code without using my own music library and database? The reason I want this is because I don't know if the changes I make will mess up the database or my current library.

**Comment by sampsyo** [comment, marked as answer]
A simple way would be to use the `-l`/`--library` flag to specify an alternative database file (and perhaps also `-d` to specify an alternative location for music files). But one thing I often use is something like `BEETSDIR=/something/testy beet` to specify an alternative configuration "root" (as opposed to the default, `~/.config/beets`).

**Reply by Bootjewolf** [reply]
Thanks!
